src/main/java/org/medibloc/panacea/PanaceaGrpcClient.java

```diff
@@ -36,6 +38,15 @@ public BaseAccount getAccount(String address) throws PanaceaApiException {
         }
     }
 
+    public Coin getBalance(String address, String denom) {
+        QueryBalanceRequest request = QueryBalanceRequest.newBuilder()
+                .setAddress(address)
+                .setDenom(denom)
+                .build();
+
+        return grpcStub.getBankQueryStub().balance(request).getBalance();
+    }
+
```

Comment:
Unlike before, the quantity of coins does not come out when inquiring by account.

Reply:
Yeah, right.... That's a stupid change from cosmos-sdk...